lib/board.rb

```diff
@@ -12,4 +12,27 @@ def render
       puts
     end
   end
+
+  # Will probably need to refactor with some helpers since this is a little long (future us problem :) )
+  # Future thought, what if we set a second parameter for either player or computer, then we can choose between
+  # an "X" or an "O" to place in the location.
```

Comment:
We probably need to pair this down a bit with some helper methods, it seems a little long.